composer.json

```diff
@@ -27,10 +29,13 @@
         "doctrine/orm": "^2.10",
         "phpstan/phpstan": "^1.0",
         "ramsey/uuid-doctrine": "^1.6",
-        "symfony/cache": "^4.4 | ^5.4 | ^6.0"
+        "symfony/cache": "^4.4 || ^5.4 || ^6.0"
```

Comment:
I've switched to `||` as it's the [Composer-recommended version](https://getcomposer.org/doc/articles/versions.md#version-range).